### PR TITLE
Added documentation for flic smart button component.

### PR DESCRIPTION
**Pull request by soldag**
**Description:**
Added documentation for flic smart button component.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#4681

### CODE REVIEW

File: source/_components/binary_sensor.flic.markdown [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+```yaml
+# Example configuration.yaml entry
+binary_sensor:
+  - platform: flic
+```
+
+Configuration variables:
+
+- **host** (*Optional*): The IP of the flic service server. (default: `localhost`)
```
Comment (Landrash) [inline source/_components/binary_sensor.flic.markdown]: Should be "IP or hostname". Example even includes the hostname of `localhost.`

**Comment by soldag** [comment]
@Landrash I updated the documentation.

**Comment by armills** [comment]
Can you include an example automation making use of the new events? See https://home-assistant.io/components/lirc/ for an example.

**Comment by soldag** [comment]
@armills Done 😉

**Comment by soldag** [comment]
@fabianbergmark I created the new PR #1556 for this.

**Pull request merged**